lib/v2/reuters/index.js

```diff
@@ -0,0 +1,50 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const navigation = ctx.params.navigation ?? 'world';
+    const rootUrl = 'https://www.reuters.com';
+    let category;
+
+    if (navigation === 'authors') {
+        category = ctx.params.category ?? 'reuters';
+    }
+    else {
+        category = ctx.params.category ?? '';
+    }
+    const currentUrl = category ? `${rootUrl}/${navigation}/${category}/` : `${rootUrl}/${navigation}/`;
+    const response = await got(currentUrl);
+    const $ = cheerio.load(response.data);
+
+    const list = $('.media-story-card__body__3tRWy a.media-story-card__heading__eqhp9')
+        .map((_, item) => {
+            item = $(item);
+            item.find('span.visually-hidden__hidden__2qXMW').remove();
+            return {
+                title: item.text(),
+                link: rootUrl + item.prop('href'),
+            };
+        })
+        .get();
```

Comment:
Still, as @TonyRL has [mentioned](https://github.com/DIYgod/RSSHub/pull/9761#issuecomment-1126986738), the route is imperfect. Basically speaking, DevTool is not so reliable if JavaScript is enabled, while `view-source:` should always be the factual criterion.

Reply:
```
view-source:https://www.reuters.com/world/
```
view-source:  can not find content text.

How do we achieve this.

Reply:
As @TonyRL has [mentioned](https://github.com/DIYgod/RSSHub/pull/9761#issuecomment-1126986738), everything you need is in `<script id="fusion-metadata" type="application/javascript">`. The easiest way to extract posts from it is using regex to match all `canonical_url` (you don't need to match the title since it can be filled while grabbing full-text in the next steps). A better way is to extract `Fusion.globalContent` using regex and then use JSON or JSON5 to parse it, but that's much harder and clumsier.

Reply:
This is kind of out of my experience.

Reply:
Or you may use [`matchAll`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/matchAll)